cdoc/job-tutorial.md

```diff
@@ -0,0 +1,268 @@
+<!--
+  Copyright (c) Microsoft Corporation
+  All rights reserved.
+
+  MIT License
+
+  Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+  documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+  the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+  to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+  The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+
+  THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+  BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+  NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+  DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+-->
+# PAI深度学习任务指南
+## 概述
+- PAI支持当前主流的深度学习框架，如CNTK、Tensorflow等。
+- 也可通过自定义Docker镜像支持其他种类的任务。
+- 用户提交job需准备一个配置文件。
+- 本文详细介绍了提交job的基本步骤。
+
+## 先决条件
+执行本文的步骤前，请确保系统已经正确部署，以及确保一个可用的Docker registry用以存储用户自定义的Docker images。
+
+## Docker镜像
+系统将在一个或多个docker容器中执行深度学习job，用户首先要根据job定制自己所需的docker镜像。
+系统提供了一个带HDFS，CUDA和cuDNN支持的基础镜像，用户可在此基础之上进行定制。
+
+首先生成基础镜像，Dockerfile示例见[Dockerfile.build.base](../Dockerfiles/cuda8.0-cudnn6/Dockerfile.build.base), 执行以下命令生成镜像:
+```sh
+docker build -f Dockerfiles/Dockerfile.build.base -t pai.build.base:hadoop2.7.2-cuda8.0-cudnn6-devel-ubuntu16.04 Dockerfiles/
+```
+
+接着，用户在构建自定义镜像时需在Dockerfile的首句添加：
+    
+    FROM pai.build.base:hadoop2.7.2-cuda8.0-cudnn6-devel-ubuntu16.04
+
+例如，要生成一个带Tensorflow的镜像，Dockerfile示例见[Dockerfile.run.tensorflow](../Dockerfiles/cuda8.0-cudnn6/Dockerfile.run.tensorflow)，执行以下命令生成：
```

Comment:
special character?

Reply:
It's normal in my vs code IDE, maybe I need to set the encoding utf-8, solved.